### PR TITLE
Pipewire polybar script

### PR DESCRIPTION
**Pull request by victortrac**

### CODE REVIEW

**Comment by jonringer** [comment]
Is there any reason to delay this? There's still no pipewire internal module https://polybar.readthedocs.io/en/latest/user/actions.html

**Comment by victortrac** [comment]
@x70b1 Would you mind reviewing?

**Review by jonringer — APPROVED** [review]
Was able to adapt this to nix use case in https://github.com/jonringer/nixpkgs-config/blob/e341f3e6a39a2bedc11f5b51fb05badf6c1aa040/nix/polybar.nix

ksnip_20211101-222728

**Comment by victortrac** [comment]
@jonringer I updated to use pamixer instead of pactl, which is a lot cleaner: https://github.com/polybar/polybar-scripts/pull/320/commits/67942681119101a295f3b8d82926adae4767881f

**Comment by patrickhener** [comment]
Any plans on merging?

**Comment by x70b1** [comment]
Sorry for the delay, I will take a look at it later.

File: polybar-scripts/pipewire/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Script: pipewire
```
Comment (x70b1) [inline polybar-scripts/pipewire/README.md:1]: Lets rename the script and directory to `pipewire-simple`.

File: polybar-scripts/pipewire/pipewire.sh [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+    action=$1
+    if [ "${action}" == "up" ]; then
+        pamixer --increase 10
+    elif [ "${action}" == "down" ]; then
+        pamixer --decrease 10
+    elif [ "${action}" == "mute" ]; then
+        pamixer --toggle-mute
+    else
+        echo " ${DEFAULT_SOURCE} |  ${VOLUME} ${DEFAULT_SINK}"
```
Comment (x70b1) [inline polybar-scripts/pipewire/pipewire.sh:17]: Please replace icons with `#` to make it easy to replace them.

File: polybar-scripts/pipewire/pipewire.sh [hunk 3]
```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+function main() {
+    DEFAULT_SOURCE=$(pw-record --list-targets | sed -n 's/^*[[:space:]]*[[:digit:]]\+: description="\(.*\)" prio=[[:digit:]]\+$/\1/p')
+    DEFAULT_SINK_ID=$(pw-play --list-targets | sed -n 's/^*[[:space:]]*\([[:digit:]]\+\):.*$/\1/p')
+    DEFAULT_SINK=$(pw-play --list-targets | sed -n 's/^*[[:space:]]*[[:digit:]]\+: description="\(.*\)" prio=[[:digit:]]\+$/\1/p')
+    VOLUME=$(pamixer --get-volume-human)
+
+    action=$1
+    if [ "${action}" == "up" ]; then
```
Comment (x70b1) [inline polybar-scripts/pipewire/pipewire.sh:10]: A `case` statement is probably a good idea here.

File: polybar-scripts/pipewire/pipewire.sh [hunk 4]
```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
```
Comment (x70b1) [inline polybar-scripts/pipewire/pipewire.sh:1]: Use `/bin/sh`. Take  also a look at the CONTRIBUTING.md

**Review by x70b1 — CHANGES_REQUESTED** [review]
@victortrac 

Nice work! 
I just added smaller notes.

**Comment by x70b1** [comment]
Awesome, thanks!

**Pull request merged**